js/components/selectedUser.js

```diff
@@ -0,0 +1,166 @@
+import UserService from "../client/userService.js";
+import Render from "./render.js";
+const apiService = new UserService();
+const render = new Render();
+
+const $deleteAllTodosBtn = document.querySelector(".clear-completed");
+const $input = document.querySelector(".new-todo");
+const $todoUl = document.querySelector(".todo-list");
+const $filters = document.querySelector(".filters");
+
+export default class SelectedUser {
+  onUserDeleteHandler = (userId) => {
+    apiService
+      .deleteUser(userId)
+      .then(() => apiService.getUsers())
+      .then(render.showUsers);
+  };
+
+  selectedUserState = (userId) => {
+    apiService.getUserTodo(userId).then(render.showItems);
+
+    $input.addEventListener("keypress", (e) => {
+      if (e.key !== "Enter") return;
+      if (e.key === "Enter") {
+        this.addNewTodo(userId, $input.value);
+        $input.value = "";
+      }
+    });
+    $deleteAllTodosBtn.addEventListener("click", () =>
+      this.deleteAllTodos(userId)
+    );
```

Comment:
```suggestion
    $deleteAllTodosBtn.addEventListener("click", () => {
      this.deleteAllTodos(userId)
    });
```
`this.deleteAllTodos(userId)`  를 return 하는 것이 아니라면 {} 중괄호를 써주는 것이 좋을 것같아요!

Reply:
코멘트 감사합니다~!ㅎㅎ